lib/core/services/user-preferences.service.ts

```diff
@@ -67,6 +68,10 @@ export class UserPreferencesService {
         }
     }
 
+    private initStoragePrefix() {
+        this.storage.storagePrefix = this.appConfig.get<string>('application.storagePrefix', '');
```

Comment:
This is something I don't really understand. We've set it already in the AlfrescoApiService. Why do we need to set it again? I assume, that the point here would be to set in only once, inside the initial bootstrapping.